sentry-samples/sentry-samples-netflix-dgs/src/main/java/io/sentry/samples/netflix/dgs/ShowsDatafetcher.java

```diff
@@ -0,0 +1,23 @@
+package io.sentry.samples.netflix.dgs;
+
+import com.netflix.graphql.dgs.DgsComponent;
+import com.netflix.graphql.dgs.DgsQuery;
+import io.sentry.samples.netflix.dgs.graphql.DgsConstants;
+import io.sentry.samples.netflix.dgs.graphql.types.Show;
+import java.util.List;
+import java.util.Random;
+
+@DgsComponent
+public class ShowsDatafetcher {
+
+  @DgsQuery(field = DgsConstants.QUERY.Shows)
+  public List<Show> shows() throws InterruptedException {
+    Thread.sleep(new Random().nextInt(500));
+
+    //        return Arrays.asList(
```

Comment:
deadcode to be removed?

Reply:
I added another data fetcher that ends with success. Thanks for pointing it out.